src/main/java/com/kodilla/ecommercee/asia/CartDto.java

```diff
@@ -0,0 +1,10 @@
+package com.kodilla.ecommercee.asia;
+
+import javax.xml.crypto.Data;
+import java.util.List;
+
```

Comment:
Pytanie: skąd taki cart? Bo w sumie nie jest to ani cart oparty na encji, ani też na endpointach w cart.spec.json (dalej nie wiem czy dobrze rozumiem ten aspekt tego projektu).  Z drugiej strony jest tu lista produktów więc pewnie jest ok.